File: Modul 5/Python/PRAK502-2310817310010-AkmallullailSyaban.py
def hitung(nilai1, nilai2):
    hitung = nilai1 - nilai2
    if hitung < 0:
        hitung = hitung * -1
    return hitung

def mutlak(nilai):
    return abs(nilai)

a, c, b, d = map(int, input().split())

hasil = hitung(a, b)+hitung(c, d)
hasil_mutlak = mutlak(hasil)
print(hasil_mutlak)

File: Modul 5/C/PRAK502-2310817310010-AkmallullailSyaban.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
    int hitunghasil = nilai1-nilai2;
    if (hitunghasil < 0){
        return hitunghasil * -1;
    }
}

int mutlak(int nilai){
    return abs(nilai);
}

int main()
{
    int a,b,c,d;
    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);
    int Hasil = hitung(a,b)+hitung(c,d);
    printf("%d",mutlak(Hasil));
    return 0;
}